__tests__/main.test.ts

```diff
@@ -121,3 +121,26 @@ test('does not display module information when it does not exist', async () => {
 
   expect(printChunkModulesTable(statsDiff)).toMatchSnapshot()
 })
+
+test('does not describe assets when requested', async () => {
+  const statsDiff = getStatsDiff(
+    await readJsonFile('./__mocks__/old-stats-assets.json'),
+    await readJsonFile('./__mocks__/new-stats-assets.json')
+  )
+
+  expect(printAssetTablesByGroup(statsDiff, 'none')).toBe('')
+})
+
+test('skips unchanged assets when requested', async () => {
```

Comment:
Could we add an explicit test for `'all'` as well, just in case the defaults change in the future?